### DISCUSSION TITLE
Telegram入群机器人提问，回答正确了为何还被踢呀？T_T

### DISCUSSION THREAD

**Discussion by 2eronia** [XTLS/Xray-core#2891] [Q&A]
问题是：
Which of the following mRNA will not be translated to arginine-valine in human mitochondria

AGAGUA
CGAGUG
CGUGUG
CGYGUU
CGGGUA
CGCGUC

我选了 CGYGUU，应该是正确答案，机器人说错，直接把我踢了 T_T

**Comment by ItsFated** [comment]
要确定所提供的mRNA序列中哪些不会在人类线粒体中被翻译成氨基酸序列精氨酸-缬氨酸（Arg-Val），我们需要考虑线粒体的遗传密码，这与标准的遗传密码略有不同。

在标准的遗传密码中：

精氨酸（Arg）由密码子CGU、CGC、CGA、CGG、AGA和AGG编码。 缬氨酸（Val）由密码子GUU、GUC、GUA和GUG编码。 然而，线粒体的遗传密码存在一些差异。例如，在人类线粒体中：

AGA和AGG，通常在标准密码中编码精氨酸，是终止密码子。 AUA，通常编码异亮氨酸，编码蛋氨酸。 现在，让我们来检查所提供的mRNA序列：

AGAGUA：在标准密码中，AGA编码精氨酸，GUA编码缬氨酸。但在线粒体密码中，AGA是终止密码子。因此，这个序列不会翻译成Arg-Val。 CGAGUG：CGA在两种密码中都编码精氨酸，GUG编码缬氨酸。 CGUGUG：CGU在两种密码中都编码精氨酸，GUG编码缬氨酸。 CGYGUU：这个序列包含一个非标准的碱基（Y）。我们需要知道具体的碱基才能确定氨基酸。 CGGGUA：CGG在两种密码中都编码精氨酸，GUA编码缬氨酸。 CGCGUC：CGC在两种密码中都编码精氨酸，GUC编码缬氨酸。 **_根据线粒体的遗传密码，序列"AGAGUA"不会在人类线粒体中被翻译成精氨酸-缬氨酸，因为在这种情况下，AGA是终止密码子。序列"CGYGUU"如果不知道"Y"代表的具体碱基，无法评估。_**

**Reply by Deleted-Account** [reply]
谢谢你的回复，我等12小时再回答一次吧

那些问题好难啊😂

**Reply by chika0801** [reply]
不如私个管理，正常你不是bot说话话，就加了

**Comment by Fangliding** [comment]
中华人民共和国知识产权行业标准 ZC 0003-2001
![image](https://github.com/XTLS/Xray-core/assets/45535409/d7e37617-c691-4aed-bbe6-aeb196121fa7)

**Reply by aoleou** [reply]
风扇哥， 麻烦帮忙通过一下入群申请，辛苦了
